dali/server/daserver.cpp

```diff
@@ -665,6 +665,7 @@ int main(int argc, const char* argv[])
 // Audit logging
         StringBuffer auditDir;
         {
+            //MORE: Does this need to change in CONTAINERIZED mode?
```

Comment:
I think it should just output to standard logging (stderr).
The audience field should be enough to discriminate it when filtering/viewing in ES etc.